.gitignore

```diff
@@ -37,7 +37,9 @@ jspm_packages
 # Optional REPL history
 .node_repl_history
 
+# Generated bundles files
 bundles/*.js
+bundles/constants/*.js
```

Comment:
just fyi - you can do `bundles/**/*.js and that will cover all subdirectories if that's what you want

Reply:
@bridiver thanks, I didn't know that.
Modified according the notice in https://github.com/brave/sync/pull/363/commits/847f24d346b2ca9ee16b2668342d70b75d3846cc , despite you already approved.